cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -450,6 +453,27 @@ func newEtcdClient(etcdServer string) (*etcd.Client, error) {
 	return client, nil
 }
 
+func newEtcdTLSClient(etcdServer, etcdCert, etcdKey, etcdCA string) (*etcd.Client, error) {
```

Comment:
the logic for TLS client seems to be more elaborate ?
do you think this is useful for the non-TLS path as well ?